src/tests/test_rest_framework_dso/test_embedding.py

```diff
@@ -0,0 +1,39 @@
+from rest_framework_dso.embedding import ObservableIterator
+
+
+class TestObservableIterator:
+    def test_final_outcome(self):
+        """Prove that the final outcome is as desired"""
+        seen1 = []
+        seen2 = []
+
+        observer = ObservableIterator("abcd", observers=[seen1.append, seen2.append])
+        assert list(observer) == ["a", "b", "c", "d"]
+
+        assert seen1 == ["a", "b", "c", "d"]
+        assert seen1 == seen2
+        assert bool(observer)
```

Comment:
Dit beantwoordt ook mijn vraag of `__bool__` nog steeds True moet geven als alle items geconsumeerd zijn :+1:

Reply:
😄